### PR TITLE
2024 09 20 uub o etiop 21

### PR DESCRIPTION
**Pull request by karljonaskarlsson**
Created ms record based on Löfgren's catalogue, the online catalogue (http://urn.kb.se/resolve?urn=urn:nbn:se:alvin:portal:record-159895), and the images available online

### CODE REVIEW

**Review by eu-genia — APPROVED** [review]
very nice, thank you

File: Uppsala/UppEt21.xml
Comment (nafisa-valieva) [inline Uppsala/UppEt21.xml:1]: Thank you! Looks like a good record to follow!

Comment (karljonaskarlsson) [reply Uppsala/UppEt21.xml:1]: If all manuscripts were as well-organized as this one, our lives would be very simple :)

File: Uppsala/UppEt21.xml [hunk 1]
```diff
@@ -0,0 +1,815 @@
...
+                            <list>
+                                <item xml:id="e1">
+                                    <desc type="StampExlibris">Paper sticker on the inside of the front board.</desc>
+                                    <q xml:lang="sv">
+                                        Uppsala Universitets<lb/>
+                                        Bibliotek<lb/>
+                                        Handskr. avd.<lb/>
+                                        O Etiop. 21<lb/>
+                                    </q>
+                                    <note>Below the rest of the text, a note in pencil: "Löfgren nr 38"</note>
```
Comment (CarstenHoffmannMarburg) [inline Uppsala/UppEt21.xml:710]: To my opinion, this note deserves its own extra.

File: Uppsala/UppEt21.xml [hunk 2]
```diff
@@ -0,0 +1,815 @@
...
+                            his mother <persName ref="PRS14571Ebrawit"/>, and
+                            his brother <persName ref="PRS14572Awkatewos"/>. The name of one further brother or sister has been erased.
+                        </origin>
+                        <provenance>
+                            At an unknown point in time, <persName ref="PRS9043Svensson" role="owner">Axel B. Svensson</persName> 
+                            aquired
+                            the manuscript. He subsequently donated the manuscript to <persName ref="PRS14565ErlingEidem" role="owner">
+                                <title>Archbishop</title>
+                                Erling Eidem</persName> on <date when="1948-09-15">15 September 1948</date>, according to
+                            a type-written note that—at least when the manuscript was catalogued by <persName ref="PRS6337Lofgren"/>—was
```
Comment (CarstenHoffmannMarburg) [inline Uppsala/UppEt21.xml:742]: I think, there must be spaces around `-` cf. 
'(...) according to a type-written note that — at least when the manuscript was catalogued by <persName ref="PRS6337Lofgren"/> — was (...)'

**Review by CarstenHoffmannMarburg — CHANGES_REQUESTED** [review]

File: Uppsala/UppEt21.xml (hunk 1 above)
Comment (nafisa-valieva) [reply Uppsala/UppEt21.xml:710]: well seen! I agree

File: Uppsala/UppEt21.xml (hunk 2 above)
Comment (karljonaskarlsson) [reply Uppsala/UppEt21.xml:742]: I think it's common English style to use em dashes here (whereas German and Swedish would have an en dash with spaces, as you suggest). Which style guide does Bm follow?

File: Uppsala/UppEt21.xml (hunk 1 above)
Comment (karljonaskarlsson) [reply Uppsala/UppEt21.xml:710]: Ok! I'll change it

File: Uppsala/UppEt21.xml (hunk 2 above)
Comment (CarstenHoffmannMarburg) [reply Uppsala/UppEt21.xml:748]: I usually check english comments with [Deepl](https://www.deepl.com/en/write#en/en-gb/according%20to%20a%20type-written%20note%20that%E2%80%94at%20least%20when%20the%20manuscript%20was%20catalogued%20by%20%3CpersName%20ref%3D%22PRS6337Lofgren%22%5C%2F%3E%E2%80%94was). The deepl-recommendation in this case is to use either commas or dashes with spaces. However, I think, it is not that important to note it in the guidelines. You may select the option of your choice.

**Review by CarstenHoffmannMarburg — APPROVED** [review]

File: Uppsala/UppEt21.xml (hunk 2 above)
Comment (eu-genia) [reply Uppsala/UppEt21.xml:748]: In Bm, as in COMSt, in Aethiopica, and in all other HLCEES publications, we should follow, if we can and care, Oxford Style guide. There should be NO spaces around the m-dashes or around the n-dashes in our publications. We have not been 100% careful in Bm as it is a work in progress, but clearly wherever you submit a catalogue or an article for print you must adhere to Oxford Style. 
https://www.aai.uni-hamburg.de/en/comst/publications/bulletin/authors.html
https://journals.sub.uni-hamburg.de/aethiopica/authorguidelines

**Pull request merged**